1-exercises/B-object-arrays/exercise-2.js

```diff
@@ -16,10 +16,11 @@ let store = [
 
 let money = 1.5;
 
-let thingsICanBuy =
-  /* todo */
+let thingsICanBuy = store
+  .filter((e) => (e.price < money ? e : null))
+  .map((e) => e.item);
```

Comment:
Good use of filter and map. Again, maybe use a more meaningful variable name instead of e.